public/config/gh-pages.js

```diff
@@ -2,46 +2,22 @@ const mapboxGlAccessToken =
   'pk.eyJ1Ijoic3RhbWVuIiwiYSI6ImNreWM1emM2NjAyNTgyb25kc2o5ZG1iMmoifQ.o2QhFbe03ilcplg9sAhYZQ';
 
 const stylePresets = [
-  {
-    id: 'mapbox-streets',
-    name: 'Mapbox Streets',
-    type: 'mapbox-gl',
-    url: 'mapbox://styles/mapbox/streets-v11',
-  },
-  {
-    id: 'mapbox-outdoors',
-    name: 'Mapbox Outdoors',
-    type: 'mapbox-gl',
-    url: 'mapbox://styles/mapbox/outdoors-v11',
-  },
-  {
-    id: 'mapbox-light',
-    name: 'Mapbox Light',
-    type: 'mapbox-gl',
-    url: 'mapbox://styles/mapbox/light-v10',
-  },
-  {
-    id: 'mapbox-dark',
-    name: 'Mapbox Dark',
-    type: 'mapbox-gl',
-    url: 'mapbox://styles/mapbox/dark-v10',
-  },
-  {
-    id: 'openstreetmap',
-    name: 'OpenStreetMap',
-    type: 'leaflet',
-    url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-    attribution:
-      '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-  },
   {
     id: 'stamen-toner',
     name: 'Stamen Toner',
-    type: 'leaflet',
+    type: 'maplibre-gl',
```

Comment:
Yes, we should switch all of our Stadia styles to be `maplibre-gl` (vector) versions instead of the raster versions, but we need to do it for all of them not just this one. And also we need to update the `url` so it's not still a `png` raster tile template.